examples/native-fungible/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Zefchain Labs, Inc.
```

Comment:
Maybe we should add a short crate documentation, and use `cargo-rdme`, too.
I don't think it needs a full description like the other examples, just a link to `fungible` and an explanation of how it's different.

Reply:
I'm doing that in the next PR with the README for this (it mostly points to `fungible`, like you said)